cmd/run/run.go

```diff
@@ -500,7 +500,8 @@ func (h *runCommandHandler) getModelNameFromArgs(models []*azuremodels.ModelSumm
 			if !model.IsChatModel() {
 				continue
 			}
-			prompt.Options = append(prompt.Options, azuremodels.FormatIdentifier(model.Publisher, model.Name))
+
+			prompt.Options = append(prompt.Options, model.ID)
```

Comment:
It's nice being able to use a simple `ID` property and not have to format an identifier from other bits.